docs/src/integrations/profiling/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Profiling"
+weight: -3
+description: Platform.sh supports a number of profiling services for optimizing your code.
+---
+
```

Comment:
So there may be some funkiness here we need to iron out. 

Right now, this is a list page, and will include those link blocks for Tideways, Blackfire and New Relic under this block of text. Since list pages don't normally have text in them, the `{{< description >}}` shortcode is automatically  placed at the top of the page if its present - so you have  duplicated text. 

So for this page (and `source-integrations/_index.md`)  I'd say either

- set `layout:single` to remove the block links. and the automatic descriptions.
- remove the duplicated first sentence from the main content and  keep everything else as is.